README.md

```diff
@@ -143,6 +145,23 @@ use `alter-table` along with `add-column` & `drop-column` to modify table level
 => ["ALTER TABLE employees DROP COLUMN address"]
 ```
 
+### create-extension
+`create-extension` can be used to create extensions with a given keyword.
+```clj
+(-> (create-extension :uuid-ossp :if-not-exists? true)
+               (sql/format :allow-dashed-names? true
```

Comment:
Could you fix the formatting here please? Sorry for nitpicking.

Reply:
Whoops! my mistake